test/e2e/helm_test.go

```diff
@@ -64,3 +65,51 @@ func TestDeclarativeHelmInvalidValuesFile(t *testing.T) {
 		Expect(SyncStatusIs(SyncStatusCodeUnknown)).
 		Expect(Condition(ApplicationConditionComparisonError, "open does-not-exist-values.yaml: no such file or directory"))
 }
+
+func TestHelmValues(t *testing.T) {
```

Comment:
I'm not sure if these tests are excessive or not. They do not sync the app, so run in 100ms.